db/migrate/20180209163253_split_post_text_into_replies.rb

```diff
@@ -0,0 +1,50 @@
+class SplitPostTextIntoReplies < ActiveRecord::Migration[5.1]
+  def up
+    Post.find_each do |post|
+      post.replies.ordered.each { |reply| reply.update_columns(reply_order: reply.reply_order+1) }
+      reply = Reply.create!(
+        post_id: post.id,
+        user_id: post.user_id,
+        character_id: post.character_id,
+        character_alias_id: post.character_alias_id,
+        icon_id: post.icon_id,
+        content: post.content,
+        created_at: post.created_at,
+        updated_at: post.updated_at,
+        skip_post_update: true
+      )
+      reply.update_columns(reply_order: 0, created_at: post.created_at)
```

Comment:
Might want to set `updated_at` to be same as the post's `edited_at` (or whatever the right one is), so we don't spawn a ton of history changes.